c2corg_ui/templates/filters.html

```diff
@@ -0,0 +1,85 @@
+<%!
+from c2corg_common.attributes import default_cultures, waypoint_types, activities
+%>
+<form app-search-filters>
+  <div class="filters">
+    <div class="row">
+      <div class="col-xs-12 col-sm-6 col-md-3 filter">
+        <label translate>Around</label>
+        <select class="form-control"><option>My position</option><option>Place</option></select>
+      </div>
+      <div class="col-xs-12 col-sm-6 col-md-3 filter">
+        <label translate>summit</label>
+        <input type="text" class="form-control">
+      </div>
+          <div class="col-xs-12 col-sm-6 col-md-3 filter">
+        <label translate>elevation</label><br>
+        <input type="range" id="elevation-filter" class="range-between">
+        <span class="slider-min-max">
+          <p class="min-value" ng-model="elevation.min">{{elevation.min}} m</p>
+          <p ng-model="elevation.max">{{elevation.max}} m</p>
+        </span>
+    </div>
+      <div class="col-xs-12 col-sm-6 col-md-3 filter">
+        <label translate>height_diff_up</label><br>
+        <input type="range" class="range-between" id="height-filter">
+        <span class="slider-min-max">
+          <p class="min-value">{{height_diff_up.min}} m</p>
+          <p>{{height_diff_up.max}} m</p>
+        </span>
+      </div>
+    </div>
+    <div class="row collapse" id="moreFilters">
+      <div class="col-xs-12 col-sm-6 col-md-3 filter">
+          <label translate>activities</label><br>
+          <button class="btn btn-default dropdown-toggle" type="button" data-toggle="dropdown">
+            <p><span translate>Select</span>
+              <span ng-if="activities.length > 0"> ({{activities.length}})</span>
+              <span class="caret"></span>
+            </p>
+          </button>
+            <ul class="dropdown-menu multiselect-box">
+              <li ng-repeat="activity in ['${"','".join(activities) |n}']">
+                <a data-value="{{activity}}" ng-click="searchFiltersCtrl.selectOption(activity, 'activities', $event)">
+                  <input type="checkbox">
+                  <span class="activity">{{mainCtrl.translate(activity) | capitalize}}</span>
+                </a>
+              </li>
+           </ul>
+      </div>
+      <div class="col-xs-12 col-sm-6 col-md-3 filter">
+        <label>Type</label><br>
+        <button class="btn btn-default dropdown-toggle" type="button" data-toggle="dropdown">
+            <p><span translate>Select</span>
```

Comment:
Incorrect indent...
Probably a lot of others. I let you check the indentation of this file.